### PR TITLE
fix(fscomponents): Move Multicarousel opacity and initialized into state

### PR DESCRIPTION
**Pull request by Cauldrath**
These values were not being updated properly when the properties changed, such as changing the images, so opacity was getting stuck set to zero.

### CODE REVIEW

**Comment by jonkoynok** [comment]
@bweissbart @skyeckstrom any chance of getting this merged? Appears to be a requirement for https://github.com/brandingbrand/dsg.rn/pull/48

File: packages/fscomponents/src/components/MultiCarousel/MultiCarousel.tsx [hunk 1]
```diff
@@ -84,24 +84,27 @@ export class MultiCarousel<ItemT> extends Component<MultiCarouselProps<ItemT>, M
     this.state = {
       currentIndex: 0,
       containerWidth: 0,
-      itemWidth: 0
+      itemWidth: 0,
+      opacity: new Animated.Value(0),
+      initialized: false
```
Comment (skyeckstrom) [inline packages/fscomponents/src/components/MultiCarousel/MultiCarousel.tsx]: This is probably the wrong way to handle the animation. We should instead use [`componentDidUpdate `](https://reactjs.org/docs/react-component.html#componentdidupdate) to set the opacity to zero and animate in whenever the image changes. We shouldn't need to keep track of `initialized` at all.

**Comment by Cauldrath** [comment]
Will now animate opacity when the image is changed. It would likely be better to animate when the image is loaded, but that would require more extensive changes to how the image references are handled.

**Comment by jasonmosleybb** [comment]
@bweissbart can you look at this?

**Comment by bweissbart** [comment]
@Cauldrath - commit linter is erroring due to "Merge from upstream" commit message.

**Comment by Cauldrath** [comment]
Squashed commits to fix history.

**Pull request merged**